channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/dto/JaicpDialerAPI.kt

```diff
@@ -105,6 +146,45 @@ class JaicpDialerAPI {
     }
 }
 
+/**
+ * Contains local time intervals by day of a week.
+ *
+ * @param mon list of time intervals on Monday
+ * @param tue list of time intervals on Tuesday
+ * @param wed list of time intervals on Wednesday
+ * @param thu list of time intervals on Thursday
+ * @param fri list of time intervals on Friday
+ * @param sat list of time intervals on Saturday
+ * @param sun list of time intervals on Sunday
+ * @param default will be used if no interval is specified for the current day of the week
+ */
+@Serializable
+data class AllowedTime(
+    val default: List<LocalTimeInterval>? = null,
+    val mon: List<LocalTimeInterval>? = null,
+    val tue: List<LocalTimeInterval>? = null,
+    val wed: List<LocalTimeInterval>? = null,
+    val thu: List<LocalTimeInterval>? = null,
+    val fri: List<LocalTimeInterval>? = null,
+    val sat: List<LocalTimeInterval>? = null,
+    val sun: List<LocalTimeInterval>? = null,
+) {
+    internal val intervals = listOf(mon, tue, wed, thu, fri, sat, sun, default)
```

Comment:
Do we really need it as a member property? Does it need to take part in a serialization process?
I think it can easily be an extension property. Furthermore, it is used in a single place, can we then completely get rid of this?

Reply:
I added this property inside the class because if the API changes, we might forget to change this list. But it really shouldn't serialize, so it's better to put it near the class.

Reply:
Yeah, I see your point, then let's keep it but as an extension function.